icu4j/maven-build/maven-icu4j/pom.xml

```diff
@@ -0,0 +1,493 @@
+    </mailingList>
+  </mailingLists>
+
+  <scm>
+    <connection>scm:git:git://github.com/unicode-org/icu.git</connection>
+    <developerConnection>scm:git:git@github.com:unicode-org/icu.git</developerConnection>
+    <url>https://github.com/unicode-org/icu</url>
+  </scm>
+
+  <issueManagement>
+    <system>JIRA</system>
+    <url>https://unicode-org.atlassian.net/projects/ICU</url>
+  </issueManagement>
+
+  <distributionManagement>
+    <repository>
+      <id>icu4j-releases</id>
+      <name>ICU4J Central Repository</name>
+      <url>https://oss.sonatype.org/service/local/staging/deploy/maven2</url>
+    </repository>
+    <snapshotRepository>
+      <id>icu4j-snapshots</id>
+      <name>ICU4J Central Development Repository</name>
+      <url>https://oss.sonatype.org/content/repositories/snapshots</url>
+    </snapshotRepository>
+  </distributionManagement>
+
+  <properties>
+    <!-- ICU4J currently builds with Java 8 sources -->
+    <maven.compiler.source>1.8</maven.compiler.source>
+    <!-- Keep the target class file versions consistent with the source file version -->
+    <maven.compiler.target>1.8</maven.compiler.target>
+    <!-- The encoding / encoding form of the source files -->
+    <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
+
+    <!-- Specification-Title -->
+    <jar.spec.title>International Components for Unicode for Java</jar.spec.title>
+    <!-- Specification-Version -->
+    <jar.spec.version>72</jar.spec.version>
+    <!-- Specification-Vendor -->
+    <jar.spec.vendor>Unicode, Inc.</jar.spec.vendor>
+
+    <!-- Implementation-Title -->
+    <jar.impl.title>International Components for Unicode for Java</jar.impl.title>
+    <!-- Implementation-Version -->
+    <jar.impl.version>72.1</jar.impl.version>
+    <!-- Implementation-Vendor -->
+    <jar.impl.vendor>Unicode, Inc.</jar.impl.vendor>
+    <!-- Implementation-Vendor-Id -->
+    <jar.impl.vendor.id>org.unicode</jar.impl.vendor.id>
+
+    <!-- Bundle-ManifestVersion -->
+    <jar.manifest.version>2</jar.manifest.version>
+    <!-- Bundle-Name -->
+    <jar.name>${project.name}</jar.name>
+    <!-- Bundle-Description -->
+    <jar.description>${project.name}</jar.description>
+    <!-- Bundle-SymbolicName -->
+    <jar.symbolic.name>${project.groupId}</jar.symbolic.name>
+    <!-- Bundle-Version -->
+    <jar.version>${project.version}</jar.version>
+    <!-- Bundle-Vendor -->
+    <jar.vendor>Unicode, Inc.</jar.vendor>
+    <!-- Bundle-Copyright -->
+    <jar.copyright.info>&#169; 2016 and later: Unicode, Inc. and others. License &#38; terms of use: http://www.unicode.org/copyright.html</jar.copyright.info>
+
+    <!-- Main-Class -->
+    <jar.main.class>com.ibm.icu.util.VersionInfo</jar.main.class>
+  </properties>
+
+  <build>
+    <plugins>
+      <plugin>
+        <groupId>org.apache.maven.plugins</groupId>
+        <artifactId>maven-compiler-plugin</artifactId>
+        <version>3.8.1</version>
```

Comment:
What you want is to move the plugin section in the parent.

If used in all the children (like the maven-compiler-plugin is), then you move it in `build/plugins`
If it is only used in some of the children then you move it in `build/pluginManagement/plugins`, with all the `executions`, `configuration` you need, and in the child that uses it you just specify the group + artifact (of the plugin)

Reply:
Done.